tests/300-db-import-export.sh

```diff
@@ -23,7 +23,7 @@ start_server() {
 }
 
 restart_server() {
-	./heketi-server --config="./heketi.json" &>> heketi.log &
+	./heketi-server --config="./heketi.json" &> heketi.log &
```

Comment:
This change does not need to be part of this PR. I'm not sure if it is in a separate patch... Reviewing on itHub makes it difficult to make that distinction.